cvat-ui/package-lock.json

```diff
@@ -1081,6 +1081,19 @@
       "resolved": "https://registry.npmjs.org/@ctrl/tinycolor/-/tinycolor-3.2.0.tgz",
       "integrity": "sha512-cP1tbXA1qJp/er2CJaO+Pbe38p7RlhV9WytUxUe79xj++Q6s/jKVvzJ9U2dF9f1/lZAdG+j94A38CsNR+uW4gw=="
     },
+    "@date-io/core": {
```

Comment:
Too many changes in this file also. I checked it out to develop, then performed query-builder installation and git stat gives me 68 changes in this file.  In your case I see 11945 changes, what is a reason?

Reply:
I don't do nothing special. Just npm i. Is it important to store auto generated file in particular structure, while generation of lock files not guaranteed same structure through different iteration?

Reply:
You do not need ``npm i`` because it updates all dependencies from ``package.json``. Instead of it you need only install one more dependency. If you with to update all, please, send it as a separated PR.  To install all dependencies from ``package-lock.json`` you need ``npm ci``

Reply:
OK